frontend/openchat-client/src/openchat.ts

```diff
@@ -2990,6 +3003,89 @@ export class OpenChat extends EventTarget {
         }
     }
 
+    private async initialStateV2(): Promise<MergedUpdatesResponse> {
+        const response = await this.api.getInitialStateV2();
+
+        const chatSummaries = (response.directChats as ChatSummary[]).concat(response.groupChats);
+
+        this.updateReadUpToStore(chatSummaries);
+
+        return {
+            wasUpdated: true,
```

Comment:
this hard-coding of `wasUpdated` to true looks a bit fishy. We don't want that to be true unless it really should be true.

Reply:
Oh yeah good point I'll fix that.